media-libs/speexdsp/speexdsp-1.2_rc3.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+inherit autotools flag-o-matic multilib-minimal
+
+MY_P=${P/_} ; MY_P=${MY_P/_p/.}
+
+DESCRIPTION="Audio compression format designed for speech -- DSP"
+HOMEPAGE="http://www.speex.org/"
+SRC_URI="http://downloads.xiph.org/releases/speex/${MY_P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~sh ~sparc ~x86 ~amd64-fbsd ~x86-fbsd ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos ~sparc-solaris ~x86-solaris"
+IUSE="static-libs cpu_flags_x86_sse cpu_flags_x86_sse2 neon"
+
+RDEPEND=""
```

Comment:
If an older version of ``media-libs/speex`` is installed, installation will fail due to file collisions between these two packages.

To resolve this, we can add a blocker: ``RDEPEND="!<media-libs/speex-1.2.0"``.

This will allow the package manager to take care of these collisions automatically.